docs/operator_parameters.md

```diff
@@ -0,0 +1,302 @@
+Postgres Operator Configuration
+===============================
+
+Postgres operator is configured via a ConfigMap defined by the
+`CONFIG_MAP_NAME` environment variable. Variable names are underscore-separated
+words.
+
+Available Configuration Parameters
+----------------------------------
+
+### General
+* **etcd_host**
+  Etcd connection string for Patroni defined as `host:port`. Not required when
+  Patroni native Kubernetes support is used. The default is empty (use
+  Kubernetes-native DCS).
+
+* **docker_image**
+  Spilo docker image for postgres instances. For production, don't rely on the
+  default image, as it might be not the most up-to-date one. Instead, build
+  your own Spilo image from the [github
+  repository](https://github.com/zalando/spilo).
+
+* **workers**
+  number of working routines the operator spawns to process requests to
+  create/update/delete/sync clusters concurrently. The default is `4`.
+
+* **max_instances**
+  operator will cap the number of instances in any managed postgres cluster up
+  to the value of this parameter. When `-1` is specified, no limits are applied.
+  The default is `-1`.
+
+* **min_instances**
+  operator will run at least the number of instances for any given postgres
+  cluster equal to the value of this parameter. When `-1` is specified, no limits
+  are applied. The default is `-1`.
+
+* **resync_period**
+  period between consecutive sync requests. The default is `5m`.
+
+### Postgres users
+* **super_username**
+  postgres `superuser` name to be created by `initdb`. The default is
+  `postgres`.
+
+* **replication_username**
+  postgres username used for replication between instances. The default is
+  `standby`.
+
+### Kubernetes resources
+* **pod_service_account_name**
+  service account used by Patroni running on individual Pods to communicate
+  with the operator. Required even if native Kubernetes support in Patroni is
+  not used, because Patroni keeps pod labels in sync with the instance role.
+  The default is `operator`.
+
+* **pod_service_account_definition**
+  The operator tries to create the pod Service Account in the namespace that
+  doesn't define such an account using the YAML definition provided by this
+  option. If not defined, a simple definition that contains only the name will
+  be used. The default is empty.
+
+* **pod_terminate_grace_period**
+  Patroni pods are [terminated
+  forcefully](https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods)
+  after this timeout. The default is `5m`.
+
+* **watched_namespace**
+  The operator watches for postgres objects in the given namespace. If not
+  specified, the value is taken from the operator namespace. A special `*`
+  value makes it watch all namespaces. The default is empty (watch the operator pod
+  namespace).
+
+* **pdb_name_format**
+  defines the template for PDB (Pod Disruption Budget) names created by the
```

Comment:
I believe here and in a few places below when we're talking about format/template it would be good to explain how to specify this format. E.g. for me it's not completely clean, if I should give it as a python format string and what parameters I can use (only cluster?).

Reply:
Good point, I've added allowed options in the template for the parameters accepting it. I think it is self evident how the strings are replaced in the template, but I am willing to add more information about it if you think it is not.